iron-list.html

```diff
@@ -209,6 +216,34 @@
       },
 
       /**
+       * When true, `iron-list` will render the items in a fixed-width grid.
+       * If true the `itemWidth` property will also need to be set
+       */
+      grid: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true
+      },
+
+      /**
+       * When `grid` is true this will define the width of each item in the
+       * grid. This plus the value of `itemOffset` will determine how many
+       * items are rendered per row
+       */
+      itemWidth: {

```

Comment:
> To take this a step further we could also grab the margin left/right and use that for the offset instead if the itemOffset is not passed.

You could, but I was afraid of the complexity that it could introduce in the code. I think if we support it for `grid`, we should also support it for `!grid`.  An alternative would be:

```html
<iron-list>
  <template>
    <div style="width: 20%">
      <div style="margin: 0 20px"> 
         [[item.name]]
      </div>
    </div>
  </template>
</iron-list>
```

but that requires an extra div and the it'd be a limitation we will have to document.